vizro-core/tests/unit/vizro/models/conftest.py

```diff
@@ -16,3 +18,22 @@ def _identity_action_function(arg=None):
 @pytest.fixture(params=[vm.Container, vm.Page])
 def model_with_layout(request):
     return request.param
+
+
+@pytest.fixture
+def mock_control_wrapper_class():
+    class MockControlWrapper(vm.VizroBaseModel):
```

Comment:
```suggestion
def MockControlWrapper():
    class _MockControlWrapper(vm.VizroBaseModel):
```

Otherwise it just looks weird at point of use.